src/routes/LandingPage/LandingPage.js

```diff
@@ -17,11 +17,11 @@ class LandingPage extends Component {
   }
 
   componentDidMount () {
-    this.props.fetchPosts()
+    // this.props.fetchPosts()
```

Comment:
??

Reply:
doesnt seem to be doing anything.

Reply:
all the posts in widgets are loaded through the group query